scripts/build_centos.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I'm fairly reluctant to include complete platform-specific build scripts here.  I'd prefer a combination of:
- good documentation (I can give you a wiki account!)
- improvements to the general CMake scripts
- binary packaging tools (i.e., ways of putting QWWAD in the "official" repos for Ubuntu, CentOS etc)

 to the general CMake build tools